.azure_pipelines/olive-ci.yaml

```diff
@@ -85,10 +87,81 @@ jobs:
 
 - template: job_templates/olive-build-template.yaml
   parameters:
-    name: Linux_GPU_CI
+    name: Linux_GPU_CI_latest_ORT
     pool: $(OLIVE_POOL_UBUNTU2004)
     windows: False
     device: gpu
+    onnxruntime: onnxruntime-gpu
+    examples:
+      bert_cuda_gpu:
+        exampleFolder: bert
+        exampleName: bert_cuda_gpu
+
+- template: job_templates/olive-build-template.yaml
+  parameters:
+    name: Linux_CI_ORT_1_15_1
+    pool: $(OLIVE_POOL_UBUNTU2004)
+    windows: False
+    onnxruntime: onnxruntime==1.15.1
```

Comment:
Seem, multiple 1.15.1. Every time we support the new ort version, we should change this CI jobs. Could we 
 extract olvie-ci.yaml as a new template?  To reduce the efforts for new ort release.

Reply:
Yes, I think we can create a new template for this yaml. 
And use it for the scheduled runs (nightly or weekly), and CI

Reply:
I choose don't user template by considering the examples might not always be same for all ORT versions.